src/main/java/com/google/sps/data/Message.java

```diff
@@ -0,0 +1,20 @@
+package com.google.sps.data;
+
+public class Message{
+    public static String sender;
+    public static String message;
+    public static Long timestamp;
```

Comment:
Looks like two separate paradigms with data models. Either make internal data publicly visible or set to private with public getters / setters